config.toml

```diff
@@ -49,6 +49,10 @@ java_fips_disabled_algorithms = [
   "HmacMD5",
 ]
 
+certified_distributions = [
```

Comment:
Pending feedback on this approach, we can start proposing these changes to various OpenShift version configs (e.g., 4.16)

Reply:
Thinking this might be better in the versioned configs, since different releases will be based on different base operating systems.

Reply:
Yes, agree :+1:

Reply:
Ack - working that feedback into this approach.

Reply:
The code to find the configuration value works with both this config and the version-specific configs. The only reason why this is still here is because the scan tests use this config in their testing.